test/unit/add_bank_account_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.